### PR TITLE
Adds Surgery tools, Service tools, Hydro tools and more into each prolathen!

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]
Adds in two new nodes!
Surgery tools for medical/sci
Most Cooking/bar/Hydro tools

[why]
QoL for people to not have to run to cargo/build a autolathen for them to make a new shaker or when they need a tool for surgery like a cautery

### CODE REVIEW

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -28,6 +28,20 @@
 	design_ids = list("robocontrol", "sflash", "borg_suit", "borg_head", "borg_chest", "borg_r_arm", "borg_l_arm", "borg_r_leg", "borg_l_leg", "borgupload",
 	"cyborgrecharger", "borg_upgrade_restart", "borg_upgrade_rename")
 
+/datum/techweb_node/surgery_tools
+	id = "surgery_tools"
+	starting_node = TRUE
+	display_name = "Basic Surgery Tools"
+	description = "Has easly printable surgery tools and drapes, thanks to Med-Co!"
```
Comment (Hatterhat) [inline code/modules/research/techweb/all_nodes.dm]:
```suggestion
	description = "Basic surgical tools, along with spare drapes in the event those get lost somewhere."
```

File: code/modules/research/techweb/all_nodes.dm [hunk 2]
```diff
@@ -28,6 +28,20 @@
...
+	design_ids = list("scalpel", "circular_saw", "surgicaldrill", "retractor", "cautery", "drapes", "hemostat", "large_beaker", "pillbottle", "syringe", "large_hypovial", "hypovial")
+
+/datum/techweb_node/basic_service
+	id = "basic_service"
+	starting_node = TRUE
+	display_name = "Stander tools for Service jobs."
```
Comment (Hatterhat) [inline code/modules/research/techweb/all_nodes.dm]:
```suggestion
	display_name = "Basic Service Tools"
```

File: code/modules/research/techweb/all_nodes.dm [hunk 3]
```diff
@@ -28,6 +28,20 @@
...
+	description = "All objects needed for the Service department to easlly do there work."
```
Comment (Hatterhat) [inline code/modules/research/techweb/all_nodes.dm]:
```suggestion
	description = "Most of, if not all, the objects needed for the Service department for their work."
```

**Comment by Ragolution** [comment]
These are all already in the Autolathe. Adding them to the Protolathe seems superfluous and a direct nerf to the autolathe, which is already niche at best. Maybe if there were improved surgical/service tools to be developed?

Having to research what can already be produced seems a little silly.

File: code/modules/research/techweb/all_nodes.dm (hunk 3 above)
Comment (nik707) [reply code/modules/research/techweb/all_nodes.dm]: > Suggested change Beta 
> [Give us feedback](https://www.research.net/r/SuggestedChanges)
> 	description = "All objects needed for the Service department to easlly do there work."
> 	description = "Most of, if not all, the objects needed by the Service department for their work."

**Comment by Ignari-Coldstorm** [comment]
I agree that the protolathe does not need those editions. Having it produce all the extra stuff cuts down on player interaction. Besides the service lathe should already be able to produce all of the botany/kitchen/bar necessary items.

**Comment by Ignari-contrib** [comment]
Code is confusing. Thought they meant the protolathe in RnD. The code apparently means the departmental lathes.

**Comment by Trilbyspaceclone** [comment]
Meanys

**Pull request closed without merging**